dashboard/test/factories/pd_factories.rb

```diff
@@ -733,6 +733,14 @@
     form_data {build(:pd_teacher1819_application_hash, program: Pd::Application::Teacher1819Application::PROGRAMS[course.to_sym]).to_json}
     application_guid nil
     regional_partner nil
+
+    trait :locked do
+      locked_at Time.now
+
+      after(:create) do |application|
+        application.update!(status: 'accepted')
+      end
```

Comment:
subtle, but since we're already forced to modify the model after creation here, how about calling `application.lock!` after updating the status, rather than setting `locked_at` directly, so it's more like how this would work in the wild?